completions/fish/README.md

```diff
@@ -0,0 +1,10 @@
+# Fish Shell completion scripts
+
+Please find here how to install `rmw` completion for [fish](https://fishshell.com/)
+
+# Installation instructions
+
+```shell
+mkdir -p ~/.config/fish/completions/ # create fish completion folder if missing
+cp rmw.fish ~/.config/fish/completions/rmw.fish
+```
```

Comment:
Please remove the surrounding backticks and indent the two code lines 4 spaces.

Reply:
I tend to use markdown backtick for shell code. But OK, no problem

Reply:
I use backticks if there's more than 4 or 5 lines of code. I never thought about it much though and don't have a specified amount.

Reply:
> I use backticks if there's more than 4 or 5 lines of code. I never thought about it much though and don't have a specified amount.

The markdown + shell will provide colorized

```
ls -la

foo bar
```

```shell
ls -la
```

```shell
$ ls -la

foo bar
```

````markdown
```
ls -la

foo bar
```

```shell
ls -la
```

```shell
$ ls -la

foo bar
```
````

Reply:
> > I use backticks if there's more than 4 or 5 lines of code. I never thought about it much though and don't have a specified amount.
> 
> The markdown + shell will provide colorized
> 
> ```
> ls -la
> 
> foo bar
> ```
> 
> ```shell
> ls -la
> ```
> 
> ```shell
> $ ls -la
> 
> foo bar
> ```
> 
> ````markdown
> ```
> ls -la
> 
> foo bar
> ```
> 
> ```shell
> ls -la
> ```
> 
> ```shell
> $ ls -la
> 
> foo bar
> ```
> ````
> 

Or not, haha. It works on gitlab.

Reply:
> I wonder now if we add an install target into the meson.build, do we really need docs that instruct a user to copy it to ~/.config/fish/completions ?

I don't think so

I will write something like:

fish completion is packaged with the binary, but if you need it, just copy `rmw.fish` into `~/.config/fish/completions/`

Reply:
I removed the block so this pointless now